include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -600,6 +600,284 @@ struct SPhysicalDeviceLimits
+        if (integerDotProductAccumulatingSaturating32BitSignedAccelerated && !_rhs.integerDotProductAccumulatingSaturating32BitSignedAccelerated) return false;
+        if (integerDotProductAccumulatingSaturating32BitMixedSignednessAccelerated && !_rhs.integerDotProductAccumulatingSaturating32BitMixedSignednessAccelerated) return false;
+        if (integerDotProductAccumulatingSaturating64BitUnsignedAccelerated && !_rhs.integerDotProductAccumulatingSaturating64BitUnsignedAccelerated) return false;
+        if (integerDotProductAccumulatingSaturating64BitSignedAccelerated && !_rhs.integerDotProductAccumulatingSaturating64BitSignedAccelerated) return false;
+        if (integerDotProductAccumulatingSaturating64BitMixedSignednessAccelerated && !_rhs.integerDotProductAccumulatingSaturating64BitMixedSignednessAccelerated) return false;
+        if (maxGeometryCount > _rhs.maxGeometryCount) return false;
+        if (maxInstanceCount > _rhs.maxInstanceCount) return false;
+        if (maxPrimitiveCount > _rhs.maxPrimitiveCount) return false;
+        if (maxPerStageDescriptorAccelerationStructures > _rhs.maxPerStageDescriptorAccelerationStructures) return false;
+        if (maxPerStageDescriptorUpdateAfterBindAccelerationStructures > _rhs.maxPerStageDescriptorUpdateAfterBindAccelerationStructures) return false;
+        if (maxDescriptorSetAccelerationStructures > _rhs.maxDescriptorSetAccelerationStructures) return false;
+        if (maxDescriptorSetUpdateAfterBindAccelerationStructures > _rhs.maxDescriptorSetUpdateAfterBindAccelerationStructures) return false;
+        if (minAccelerationStructureScratchOffsetAlignment > _rhs.minAccelerationStructureScratchOffsetAlignment) return false;
+        if (variableSampleLocations && !_rhs.variableSampleLocations) return false;
+        if (sampleLocationSubPixelBits > _rhs.sampleLocationSubPixelBits) return false;
+        if (!_rhs.sampleLocationSampleCounts.hasFlags(sampleLocationSampleCounts)) return false;
+        if (maxSampleLocationGridSize.width > _rhs.maxSampleLocationGridSize.width) return false;
+        if (maxSampleLocationGridSize.height > _rhs.maxSampleLocationGridSize.height) return false;
+        if (sampleLocationCoordinateRange[0] < _rhs.sampleLocationCoordinateRange[0] || sampleLocationCoordinateRange[1] > _rhs.sampleLocationCoordinateRange[1]) return false;
+        if (minImportedHostPointerAlignment > _rhs.minImportedHostPointerAlignment) return false;
+        if (minFragmentDensityTexelSize.width < _rhs.minFragmentDensityTexelSize.width) return false;
+        if (minFragmentDensityTexelSize.height < _rhs.minFragmentDensityTexelSize.height) return false;
+        if (maxFragmentDensityTexelSize.width > _rhs.maxFragmentDensityTexelSize.width) return false;
+        if (maxFragmentDensityTexelSize.height > _rhs.maxFragmentDensityTexelSize.height) return false;
```

Comment:
Can you put a `// TODO: Revise min/maxFragmentDensityTexelSize comparisons` 
I'm not sure how it's best to compare these, and these should work because all I can see in the reports is either `32x32` or `256x256` which should work right now. they're all squares